### PR TITLE
Update pin

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Apparently this is the preferred method of pinning

**Comment by timkpaine** [comment]
I've been through this a bunch of times, at least they're finally documenting what is preferred.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -55,7 +56,7 @@ outputs:
         - python
       run:
         - python
-        - {{ pin_subpackage("arrow_odbc", exact=True) }}
+        - {{ version }}*_{{ build_num }}
```
Comment (timkpaine) [inline recipe/meta.yaml:59]: Package name?

Comment (dhirschfeld) [reply recipe/meta.yaml:59]: 🤦

**Comment by dhirschfeld** [comment]
> I've been through this a bunch of times, at least they're finally documenting what is preferred.

It was news to me, but I did run into issues with the hash when when I packaged `solara`, and had to instead pin the version/build there to fix it.

**Comment by dhirschfeld** [comment]
Just running a test...

**Comment by dhirschfeld** [comment]
This should be good to merge I guess.

**Review by pacman82 — APPROVED** [review]
Thanks for caring! LGTM

**Pull request merged**